tools/config/__init__.py

```diff
@@ -498,6 +498,24 @@ def __init__(self, tgt, top_level_dirs=None, app_config=None):
         self.cumulative_overrides = {key: ConfigCumulativeOverride(key)
                                      for key in CUMULATIVE_ATTRIBUTES}
 
+        if symbols:
+            if not self.app_config_location:
+                self.app_config_location = "."
+            self.app_config_data.setdefault("target_overrides", {})
+            self.app_config_data["target_overrides"].setdefault(
+                self.target.name, {})
+            overrides = self.app_config_data["target_overrides"][self.target.name]
+            for symbol in symbols:
+                try:
+                    name, value = symbol.split("=")
+                except ValueError:
+                    continue
```

Comment:
Should there be some sort of message here, indicating that something wasn't specified correctly?

Reply:
Nope. It's a macro it it's not a config param.